src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -93,6 +93,24 @@ TEST_F(TransformationTestsF, AssertAndStringTensors) {
 }
 
 TEST_F(TransformationTestsF, UnsortedNodes) {
-    { function = convert_model("forward_edge_model_unsorted/forward_edge_model_unsorted.pb"); }
-    { function_ref = convert_model("forward_edge_model/forward_edge_model.pb"); }
+    { model = convert_model("forward_edge_model_unsorted/forward_edge_model_unsorted.pb"); }
+    { model_ref = convert_model("forward_edge_model/forward_edge_model.pb"); }
+}
+
+TEST_F(TransformationTestsF, ModelWithSwishF32BodyGraph) {
+    {
+        model = convert_model("swish_f32/swish_f32.pb");
```

Comment:
We need to also test the case when there are 2 or more instances of `swish_f32` in the model. So that we check that model injection works as expected.

Reply:
We have such a case in the EfficientDet model from e2e scope. I want this to be included into e2e scope.